src/cloudformation_cli_python_lib/resource.py

```diff
@@ -46,9 +46,9 @@ def wrapper(self: Any, event: MutableMapping[str, Any], context: Any) -> Any:
         try:
             response = entrypoint(self, event, context)
             serialized = json.dumps(response, cls=KitchenSinkEncoder)
-        except Exception as e:  # pylint: disable=broad-except
```

Comment:
curious why this has been changed? Isn't it better if we give more context about the exception?